trishalanglois/refactor-tractor
# Engl tags for html elements

#### What's this PR do?
Added english language tags to all elements on the DOM and added through innerHTML that a screen reader would read to a user.
#### Where should the reviewer start?
Review the index.html and index.js files to see changes on all text elements, including paragraph and headings.  Index.html will include the elements that are on the DOM at time of window load, and index.js includes all the innerHTML elements.
#### How should this be manually tested?
N/A -- we'll need to run an ARIA accessibility test to see if it picks up the changes (and there are lots more accessibility changes to come)
#### Any background context you want to provide?
As previously mentioned, there will be more accessibility changes to come, e.g. text color, alt tags for images, etc.
#### What are the relevant tickets?
N/A -- closed issue already. I'll wait until making a PR to do it next time so I can do the cool trick where you close the issue with the PR.
#### Screenshots (if appropriate)
N/A
#### Questions:
- Do either of you have a favorite accessibility screener we all want to use to be consistent?

@brittanystoroz
@khalidwilliams
